src/main/java/picard/cmdline/CommandLineParser.java

```diff
@@ -235,9 +238,9 @@ private String getUsagePreamble() {
 
         if (null != this.programVersion && 0 < this.programVersion.length()) {
             usagePreamble += "Version: " + getVersion() + "\n";
+            usagePreamble += "HTSJDK Version: " + getHtsjdkVersion() + "\n";
```

Comment:
Instead of adding calls to getHtsjdkVersion everywhere that getVersion is called, I would change getVersion() to return a string that includes both the picard version and the htsjdk version and expose a separate getPicardVersion method that does what the old getVersion did. Otherwise it will end up being inconsistently reported.